### PR TITLE
refactor(sdk-1723): remove no-op code

### PR DESCRIPTION
**Pull request by JagadeeshKaricherla-branch**
Remove no-op code

# Pull Request Template

## Description

remove no-op code for the following deprecated methods :

- applyCode
- getCode
- credits
- creditHistory
- redeem
- autoAppIndex

Fixes # (issue) : sdk-1723

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit test
- [ ] Integration test

## JS Budget Check

Please mention the size in kb before abd after this PR

| Files            | Before      | After       | 
| -----------      | ----------- | ----------- |
| dist/build.js.   |             |             |
| dist/build.min.js|             |             |

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Mentions: 
List the person or team responsible for reviewing proposed changes.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

**Comment by gdeluna-branch** [comment]
Just confirming that this old PR https://github.com/BranchMetrics/web-branch-deep-linking-attribution/pull/788 is obsolete?

File: src/onpage.js [hunk 1]
```diff
@@ -34,29 +34,21 @@
 	},
 	[
 		'addListener',
-		'applyCode',
-		'autoAppIndex',
 		'banner',
 		'closeBanner',
 		'closeJourney',
-		'creditHistory',
 		'credits',
```
Comment (gdeluna-branch) [inline src/onpage.js:40]: Is this `credits` string still needed?

Comment (gdeluna-branch) [reply src/onpage.js:40]: Looks deleted actually https://github.com/BranchMetrics/web-branch-deep-linking-attribution/pull/926/files#diff-cf134fe4c8883dd92eef5f2b91797d08dd1ffed8e57ee3c39e794bdf93e3940dL1584

File: test/6_branch.js [hunk 2]
```diff
@@ -835,17 +835,6 @@ describe('Branch', function() {
 		});
 	});
 
-	describe('sendSMS', function() {
```
Comment (gdeluna-branch) [inline test/6_branch.js:838]: There's a reference to SMS in https://github.com/BranchMetrics/web-branch-deep-linking-attribution/blob/master/docs/1_intro.md?plain=1#L23

File: src/onpage.js (hunk 1 above)
Comment (JagadeeshKaricherla-branch) [reply src/onpage.js:40]: good catch. Not needed.

File: test/6_branch.js (hunk 2 above)
Comment (JagadeeshKaricherla-branch) [reply test/6_branch.js:838]: Fixed.Thanks.

**Review by gdeluna-branch — APPROVED** [review]

**Pull request merged**